desktop/install/archlinux.md

```diff
@@ -6,7 +6,7 @@ redirect_from:
   - /desktop/linux/install/archlinux/
 ---
 
-This topic discusses installation of Docker Desktop from an [Arch package](https://desktop-stage.docker.com/linux/main/amd64/docker-desktop-4.19.0-x86_64.pkg.tar.zst?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64) that Docker provides in addition to the supported platforms. Docker has not tested or verified the installation.
+This topic discusses installation of Docker Desktop from an [Arch package](https://desktop-stage.docker.com/linux/main/amd64/docker-desktop-4.19.0-x86_64.pkg.tar.zst?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64) that Docker provides in addition to the supported platforms. Docker hasn't tested or verified the installation.
```

Comment:
Do we know why this is referring to the "desktop-stage" domain? Looks like that probably shouldn't be documented in our docs 🤔

Reply:
Yeah, it’s the stage bucket, URLs have a 30 days decay time there…
We should use https://desktop.docker.com/linux/main/amd64/106363/docker-desktop-4.19.0-x86_64.pkg.tar.zst instead.

Reply:
@aevesdocker Could you update the URL?